styles.css

```diff
@@ -0,0 +1,73 @@
+@media (max-width:780px){
+  #navbar {
+    background-color: maroon;
+    height: 40px;
+    display: flex;
+    justify-content: flex-end;
+    position: fixed;
+    width: 100%
+    
+  }  
+  .navContent{
+    margin: 13px;
+    color: white;
+    text-decoration: none;
+    font-family: Sans-serif, Abadi;
+    font-size: 18;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  
+  }
+  #welcome-section{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    height: 100vh;
+    background-color: rgb(0,0,0,0.9);
+    
```

Comment:
Please be careful of the open lines before closing brackets there should however be a an open line after the brackets for readability.